src/main.qml

```diff
@@ -185,6 +193,7 @@ Application {
         triggeredOnStart: true
         onTriggered: {
             if(seconds <= 0) {
+                notificationSound.play()
```

Comment:
Unfortunately this won't work when the app is closed.

Reply:
ok will add the code into the alarm presenter ... will remove these commits make a new PR